pysipp/report.py

```diff
@@ -46,12 +46,22 @@ def emit_logfiles(agents2procs, level='warn', max_lines=100):
     """
     emit = getattr(log, level)
     for ua, proc in agents2procs:
+        i=5
```

Comment:
In fact, I do not know if the loop is useful. As the occurrence of exceptions is rare, I do not know under what conditions they occur. The loop gives some time to STREAM to be created, but perhaps instead it is already destroyed, or will not be built.
With this change, the exceptions have disappeared.